transcom/mymove
# B 20526 int - Backend | Sort Customer dropdown of Counseling Offices

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20526)

## Summary
The list from which the customer makes their counseling office selection will be sorted by nearest to farthest from origin duty location.

### How to test

1. Login as a customer, create a move with a non-primed counseled duty location (Scott AFB, IL 62225)
2. Verify the counseling options presented are displayed from closest to furthest distance and in accordance with the names in the screenshot below.


![ScottAFBCounselingOfficeDistances](https://github.com/user-attachments/assets/db190503-52a3-4c93-90df-d7897fa170fe)
